### PR TITLE
Simplify web socket client

### PR DESCRIPTION
**Pull request by DmytroOrlov**

### CODE REVIEW

File: server/app/views/templates/mainFlashToUser.scala.html [hunk 1]
```diff
@@ -3,6 +3,6 @@
 @(title: String)(content: Html)(implicit flash: Flash, msg: Messages, session: Session)
 
 @main(title) {
-    <div class="flashToUser">@flash.get(controllers.flashToUser).getOrElse("")</div>
+    @flash.get(controllers.flashToUser).map { f => <div class="flashToUser">@f</div> }
```
Comment (novakov-alexey) [inline server/app/views/templates/mainFlashToUser.scala.html]: did you check whether html view remains the same when div tag is not always printed?

Comment (DmytroOrlov) [inline server/app/views/templates/mainFlashToUser.scala.html]: It seems that web-app view remains the same, which additional checks do we need?

Comment (novakov-alexey) [inline server/app/views/templates/mainFlashToUser.scala.html]: Yes, looks fine

**Comment by novakov-alexey** [comment]
There is one problem at AddTask form. If I put some fields with code and submit the form having some of the fields empty, then previously filed fields are not filled and user needs to fill them again.

**Comment by novakov-alexey** [comment]
Can you check why do we lose filled values? Maybe after some refactoring this feature was broken

**Comment by novakov-alexey** [comment]
We can merge this, but need to check form does not keep values on submit

**Comment by DmytroOrlov** [comment]
fine, let's merge and create separate task

**Comment by DmytroOrlov** [comment]
> There is one problem at AddTask form. If I put some fields with code and submit the form having some of the fields empty, then previously filed fields are not filled and user needs to fill them again.

This problem can present in this file from very beginning. We do `addTaskForm.bindFromRequest` twice instead of introduce _form validation_ and use error handler:
```
addTaskForm.bindFromRequest.fold(
      errorForm => {
        Future.successful(BadRequest(views.html.addTask(errorForm)))
      }
```

**Comment by novakov-alexey** [comment]
Where is it done twice?

**Comment by novakov-alexey** [comment]
Where is second call? I am not sure I understood what you mean

**Pull request merged**